test/lib/server/routes.test.js

```diff
@@ -29,5 +31,13 @@ module.exports = () => {
                 assert(apiDocumentationCotroller.apiDocsJson.length === routes.length);
             });
         });
+        describe('Endpoints', () => {
+            routes.map(async ({ path }) => {
+                const url = `${config.http.tls ? 'https' : 'http'}://localhost:${config.http.port}/api${replaceAll(path, /:[^/]+/, '0')}`;
+                it(`Check if endpoint ${path} parametrized as ${url} works correctly`, async () => {
```

Comment:
Please stick to the convention:
```
describe('Endpoints', ...
    it('Should map endpoint path to url correctly', ...
```

Reply:
it doesn't map, map is used for creating series of `it` instances

Reply:
it checks if endpoint works...